commit df6d78ba8b383bca4a6c6b8a60c01bbd3bfffde5
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-06-03T13:34:05-04:00 (4 months ago)
    
    TRITON-1714 update cnapi base image to triton-origin-x86_64-18.4.0
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/Makefile b/Makefile
index a3fb985..9932127 100644
--- a/Makefile
+++ b/Makefile
@@ -44,10 +44,9 @@ JSSTYLE_FLAGS	= -o indent=4,doxygen,unparenthesized-return=0
 SMF_MANIFESTS_IN = smf/manifests/cnapi.xml.in
 
 NODE_PREBUILT_VERSION=v6.17.0
-NODE_PREBUILT_TAG=zone
+NODE_PREBUILT_TAG=zone64
 ifeq ($(shell uname -s),SunOS)
-	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
-	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
+	NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 endif
 
 COAL ?= root@10.99.99.7
@@ -75,7 +74,8 @@ ROOT            := $(shell pwd)
 RELEASE_TARBALL := $(NAME)-pkg-$(STAMP).tar.gz
 RELSTAGEDIR          := /tmp/$(NAME)-$(STAMP)
 
-BASE_IMAGE_UUID = 04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f
+# our base image is triton-origin-x86_64-18.4.0
+BASE_IMAGE_UUID = a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME = $(NAME)
 BUILDIMAGE_DESC	= SDC CNAPI
 AGENTS		= amon config registrar
diff --git a/deps/eng b/deps/eng
index d25b8fc..4e33bf4 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit d25b8fc60fb8c649559924870fe3aaf75e7421d5
+Subproject commit 4e33bf4e26bc6cc8d93b3851fed97c11f513bfe4
